ModelBundle/Repository/NodeRepository.php

```diff
@@ -189,6 +190,22 @@ public function findByParentIdAndSiteId($parentId, $siteId)
     }
 
     /**
+     * @param string $path
+     * @param string $siteId
+     *
+     * @throws \Doctrine\ODM\MongoDB\MongoDBException
+     *
+     * @return mixed
+     */
+    public function findByPathAndSiteId($path, $siteId)
```

Comment:
The naming rule should be findOne or findAll in my mind. So I think the first name was confusing too.

Do we have somewhere the general renaming 'normalization' rules created by @nbouquet ?